### PR TITLE
Track symbol index instead of symbol name

### PR DESCRIPTION
**Pull request by asam01**
Changes to makeInvestment and getCurrentPrice to account for symbol tracking by index instead of name.
Minor style fixes throughout.

### CODE REVIEW

File: src/Components/firestore-access.js [hunk 1]
```diff
@@ -212,42 +213,47 @@ export const getDate = async (db, roomID) => {
...
+
+  return prices;
 }
 
 export const advanceDay = async (db, roomID) => {
   const roomRef = await db.collection('Rooms').doc(roomID);
   const data = await roomRef.get();
   const roomData = data.data();
+
   if(roomData.day_index < roomData.dates.length){
```
Comment (John42506176Linux) [inline src/Components/firestore-access.js]:
```suggestion
  if(roomData.day_index < roomData.dates.length) {
```

Comment (John42506176Linux) [reply src/Components/firestore-access.js]: If you want to do the space thing then might as well be consistent.

File: src/Components/firestore-access.js [hunk 2]
```diff
@@ -178,28 +176,31 @@ export const initializeQuiz = async (symbols, roomId, periodLen, endDates) => {
...
+    });
   }
-  catch(error){
-    console.log("Error is " +  error)
+  catch(error) {
+    console.log("Error is " +  error);
   }
 }
+
 export const getSymbols = async (db,roomID) =>{
```
Comment (John42506176Linux) [inline src/Components/firestore-access.js]:
```suggestion
export const getSymbols = async (db,roomID) => {
```

File: src/Components/firestore-access.js [hunk 3]
```diff
@@ -127,8 +125,8 @@ export const initSymbols = async(db,Industry,Sector,NumOfSymbols) =>{
     catch(error){
       console.log("Error with Query: " + error)
     }
-
   }
+
   else if(Industry !== null){
```
Comment (John42506176Linux) [inline src/Components/firestore-access.js]:
```suggestion
  else if(Industry !== null) {
```

File: src/Components/firestore-access.js [hunk 4]
```diff
@@ -127,8 +125,8 @@ export const initSymbols = async(db,Industry,Sector,NumOfSymbols) =>{
     catch(error){
```
Comment (John42506176Linux) [inline src/Components/firestore-access.js]:
```suggestion
    catch(error) {
```

**Review by John42506176Linux — CHANGES_REQUESTED** [review]
I just went over it looks good just add the changes I left there.

**Review by John42506176Linux — APPROVED** [review]
Looks good

File: src/Components/firestore-access.js [hunk 5]
```diff
@@ -87,120 +87,128 @@ export const initDates = async (db, symbols, Rounds) => {
...
+
   const datesD ={
     "dates" : dates,
     "period": random_period
   }
-  return datesD
+  return datesD;
 }
 
-export const initSymbols = async(db,Industry,Sector,NumOfSymbols) =>{
```
Comment (yukyhaha) [inline src/Components/firestore-access.js:106]: please add some comments for each export function and explain what does this function do

**Review by yukyhaha — APPROVED** [review]

File: src/Components/firestore-access.js (hunk 5 above)
Comment (asam01) [reply src/Components/firestore-access.js:106]: just added!

**Pull request merged**